src/compile/axis/parse.ts

```diff
@@ -167,7 +167,7 @@ function mergeAxisComponent(merged: AxisComponent, child: AxisComponent): AxisCo
   return merged;
 }
 
-function isExplicit<T extends string | number | boolean | object>(
+function isExplicit<T extends string | number | boolean | Dict<any>>(
```

Comment:
```suggestion
function isExplicit<T extends string | number | boolean | Dict<unknown>>(
```

We should be using unknown, not any here. That was what the lint warning was about, right?